### PR TITLE
Remove app path prefix

### PR DESCRIPTION
**Pull request by feedmypixel**

### CODE REVIEW

**Review by flurdy — APPROVED** [review]

File: src/config/nunjucks/context/index.js [hunk 1]
```diff
@@ -24,16 +23,13 @@ try {
...
     navigation: buildNavigation(request),
     getAssetPath: function (asset) {
       const webpackAssetPath = webpackManifest[asset]
 
-      if (!appPathPrefix || appPathPrefix === '/') {
-        return `${assetPath}/${webpackAssetPath}`
-      } else {
-        return `${appPathPrefix}${assetPath}/${webpackAssetPath}`
-      }
+      return `${assetPath}/${webpackAssetPath}`
```
Comment (chotai) [inline src/config/nunjucks/context/index.js:32]: worth inlining? 

```
return `${assetPath}/${webpackManifest[asset]}`
```

File: src/server/about/controller.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    return h.view('about/index', {
+      pageTitle: 'About',
+      heading: 'About',
+      breadcrumbs: [
+        {
+          text: 'Home',
+          href: '/'
+        },
+        {
+          text: 'About'
```
Comment (chotai) [inline src/server/about/controller.js:12]: should we add a href?

**Review by chotai — APPROVED** [review]
LGTM

File: src/server/about/controller.js (hunk 2 above)
Comment (feedmypixel) [reply src/server/about/controller.js:12]: no the final breadcrumb doesn't have a link

**Pull request merged**